ctms/crud.py

```diff
@@ -45,15 +45,16 @@
 
 def ping(db: Session):
     try:
-        db.execute(select([func.now()])).first()
```

Comment:
This statement threw some Sqlalchemy error, and `SELECT 1` does the same job of "asserting" that we can make queries and retrieve results